acceptance/tests/direct_puppet/catalog_uuid_correlates_catalogs_with_reports.rb

```diff
@@ -1,4 +1,9 @@
 test_name "PUP-5872: catalog_uuid correlates catalogs with reports" do
+
+  tag 'audit:medium',
+      'audit:acceptance',
+      'server'
```

Comment:
Yes, and in doing so perform an end-to-end validation that cached catalogs behave properly and retain the expected uuid between the server report and the applied agent catalog. Does this not map to a server based acceptance test?